cli-feed-v3.json

```diff
@@ -989,7 +989,46 @@
         }
       ]
     },
-    "1.0.11": {
+    "2.18.4": {
+            "Microsoft.NET.Sdk.Functions": "1.0.24",
+            "cli": "https://functionscdn.azureedge.net/public/2.4.498/Azure.Functions.Cli.min.win-x86.2.4.498.zip",
+            "sha2": "0610BF9B70168836BD7D3F6FEE4689890259BBF2F8CFBAB3C41BF60B09CC3E2B",

```

Comment:
I manually copied in the sha2 values from the build files for all 5 of these

Reply:
You'd think a nice tool can just take the release number and auto generate these entries.........................................................